app/routeMap.js

```diff
@@ -13,6 +13,7 @@ let discourse = require('./discourse')
 
 module.exports = {
   '/': explore,
+  '/explore': explore,
```

Comment:
In case root view changes away from explore (say it goes to a splash screen instead). 
So that after a user deletes a project I can router.goTo('/explore') instead of goTo('/'), which would create a weird flow if '/' no longer points to explore.